projects/dsp-ui/src/lib/action/pipes/formatting/formatted-boolean.pipe.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { FormattedBooleanPipe } from './formatted-boolean.pipe';
+
+describe('FormattedBooleanPipe', () => {
+    let pipe: FormattedBooleanPipe;
+
+    beforeEach(() => {
+        pipe = new FormattedBooleanPipe();
+    });
+
+    it('create an instance', () => {
+        expect(pipe).toBeTruthy();
+    });
+
+    it('should return "true" (no format specified)', () => {
+        const myBoolean = true;
+
+        const convertedBoolean = pipe.transform(myBoolean);
+
+        expect(convertedBoolean).toEqual('true');
+    });
+
+    it('should return "false" (format specified)', () => {
+        const myBoolean = false;
+
+        const convertedBoolean = pipe.transform(myBoolean, 'true-false');
+
+        expect(convertedBoolean).toEqual('false');
+    });
+
+    it('should return "yes"', () => {
+        const myBoolean = true;
+
+        const convertedBoolean = pipe.transform(myBoolean, 'yes-no');
+
+        expect(convertedBoolean).toEqual('yes');
+    });
+
+    it('should return "off"', () => {
+        const myBoolean = false;
+
+        const convertedBoolean = pipe.transform(myBoolean, 'on-off');
+
+        expect(convertedBoolean).toEqual('off');
+    });
```

Comment:
are the `displayTypes` in `FormattedBooleanPipe` ever used?

Reply:
not yet, but the option has been implemented for later when the ontology editor is made

Reply:
@tobiasschweizer actually, I did use it in the boolean value template, but it's defaulting to true/false because I didn't specify a displayType

https://github.com/dasch-swiss/knora-ui-ng-lib/blob/42c73f4895bc3f6efb202417361b92e447ea929b/projects/dsp-ui/src/lib/viewer/values/boolean-value/boolean-value.component.html#L2